.gitignore

```diff
@@ -1,9 +1,3 @@
-# Ignore all hidden files
-.*
```

Comment:
Not sure this is preferable to allowing .flake8 with a `!` in gitignore

Reply:
As you wish